etc/keepass.profile

```diff
@@ -1,6 +1,9 @@
 # keepass password manager profile
-noblacklist ${HOME}/.config/keepass
 noblacklist ${HOME}/.keepass
+noblacklist ${HOME}/.config/keepass
+noblacklist ${HOME}/.config/KeePass
+noblacklist ${HOME}/*.kdbx
+noblacklist ${HOME}/*.kdb
```

Comment:
You are right, Keepass 1 can't open .kdbx files. From a security standpoint the way I suggested it is clearly not ideal. 
But my idea was that maybe not all distributors name their executables the way firejail names their keepass profiles, so I took here rather a functional standpoint.